kibbeh/src/ui/Tag.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+interface TagProps { }
+
+export const Tag: React.FC<TagProps> = ({
+  children,
+}) => {
+  return (
+    <div
+      className={`bg-primary-600 select-none text-xs px-1 font-bold text-primary-100 justify-center items-center rounded`}
```

Comment:
Background should be `gray-700` and `gray-600` on hover. Padding should be `px-2` on both the flag tag and the text flag.